cmd/snap-update-ns/user.go

```diff
@@ -72,7 +80,8 @@ func (ctx *UserProfileUpdateContext) LoadDesiredProfile() (*osutil.MountProfile,
 }
 
 func applyUserFstab(snapName string) error {
-	ctx := NewUserProfileUpdateContext(snapName, os.Getuid())
+	fromSnapConfine := true // currently always true, no persistence yet.
+	ctx := NewUserProfileUpdateContext(snapName, fromSnapConfine, os.Getuid())
```

Comment:
I think we should do `Lock()` and `defer unlock()` here even if they are stubs/no-ops to, so that high-level semantics are visible already.

Reply:
Don't worry, we will, just not yet in this patch. This is coming up in https://github.com/zyga/snapd/commit/b3e0944ffb3d2d90aeeadca39c3941f90ff305ca